kubernetes/charts/edge-kubernetes/templates/_helpers.tpl

```diff
@@ -140,8 +143,31 @@ proxy:
   config_path: "/etc/iotedge-proxy"
   trust_bundle_config_map_name: "iotedged-proxy-trust-bundle"
   trust_bundle_path: "/etc/trust-bundle"
-  memory_limit: {{ .Values.iotedgedProxy.limits.memoryLimit | quote }}
-  cpu_limit: {{ .Values.iotedgedProxy.limits.cpuLimit | quote }}
+  resources:
+{{ toYaml .Values.iotedgedProxy.resources | indent 4 }}
+
+{{ end }}
+
+{{/* Template for iotedged's configuration YAML. */}}
+{{- define "edge-kubernetes.agentconfig" }}
+AgentConfigPath: "/etc/edgeAgent"
```

Comment:
The primary idea was to have these constants defined once in the Helm chart, because they are shared by iotedged and Agent.

Changing the AgentConfigPath would not work without other code changes. However, the AgentConfigVolume can be modified and work just fine. The AgentConfigMapName could also be parameterized through additional Helm chart updates. 

We haven't exposed them for configuration by the customer, but the way this was done allows it. I intentionally left this open to allow for resolving naming conflicts.